workers/unity/Assets/Playground/Scripts/Worker/MobileClientWorkerConnector.cs

```diff
@@ -1,20 +1,57 @@
+using System;
 using Improbable.Gdk.Core;
 using Improbable.Gdk.Mobile;
+using Improbable.Worker.CInterop;
 using UnityEngine;
 
 namespace Playground
 {
-    public class MobileClientWorkerConnector : DefaultMobileWorkerConnector
+    public class MobileClientWorkerConnector : WorkerConnector, MobileConnectionFlowInitializer.IMobileSettingsProvider
     {
 #pragma warning disable 649
         [SerializeField] private GameObject level;
+        [SerializeField] private string ipAddress;
 #pragma warning restore 649
 
         private GameObject levelInstance;
 
         public async void Start()
         {
-            await Connect(WorkerUtils.MobileClient, new ForwardingDispatcher()).ConfigureAwait(false);
+            var connParams = CreateConnectionParameters(WorkerUtils.MobileClient);
+            connParams.Network.UseExternalIp = true;
+            connParams.Network.ConnectionType = NetworkConnectionType.Kcp;
```

Comment:
Should we read this from commandline?

Reply:
if you did that, wouldn't you have to read in the parameters for it too?

Reply:
This relates to the "Unresolved Question" from the description: 

>  How to handle connection parameters? Currently they are constructed and modified directly by the user. Do we want to provide a similar mechanism for reusable blobs of code to modify them.

In any case - its left up to the user. I want to avoid magic behavior that isn't opt-in. We can certainly provide a `ConnectionParameters.FromCommandLine()` or similar that has a reasonable implementation.

Reply:
Discussed offline with @zeroZshadow - going to implement a similar pattern to the connection flow initializers so you have something like: 

```csharp
var connParams = CreateConnectionParameters(WorkerUtils.MobileClient, new CommandLineConnectionParametersInitializer());
```

What a mouthful.